openssl/src/crypto/pkey.rs

```diff
@@ -69,6 +70,22 @@ impl PKey {
         }
     }
 
+    /// Reads certificate from PEM, takes ownership of handle
+    pub fn private_key_from_pem<R>(reader: &mut R) -> Result<PKey, SslError> where R: Read {
```

Comment:
Because there could also be a method `public_key_from_pem` and I thought it would be good to have am unambiguous name for the function.